include/ost-sampleconfig.php

```diff
@@ -47,6 +47,9 @@
 # Table prefix
 define('TABLE_PREFIX','%CONFIG-PREFIX');
 
+#Dev switch for disabling autocron
+define('DEV', CFG::DEV);
```

Comment:
Whoops. Should be false on sample config. I`ll fix that tomorrow
25.2.2015 17.03 kirjoitti "Jared Hancock" <notifications@github.com>:

> In include/ost-sampleconfig.php
> <https://github.com/osTicket/osTicket-1.8/pull/1755#discussion_r25345643>:
>
> > @@ -47,6 +47,9 @@
> >  # Table prefix
> >  define('TABLE_PREFIX','%CONFIG-PREFIX');
> >
> > +#Dev switch for disabling autocron
> > +define('DEV', CFG::DEV);
>
> Where is CFG::DEV defined?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/osTicket/osTicket-1.8/pull/1755/files#r25345643>.
>